safe-global/safe-wallet-web
# Analytics: update Privacy policy for Spindl + refactor legal pages

## What it solves

I've cleaned up the components folder a little bit and remove unnecessary directories.

Also updated the Privacy Policy to include information about Spindl and the tracking of signer addresses.

## How to test it

Just click through pages like Terms, Imprint, Cookie policy etc and check that nothing has changed visually.
